docs/changelog.rst

```diff
@@ -3,4 +3,4 @@
 Changelog
 =========
 
-.. include:: ../CHANGELOG.rst
+.. include:: ../CHANGELOG.md
```

Comment:
Did you build the docs on this one to confirm it will look okay? I considered updating this a while back but when I searched online it seems like the rst file won't render the markdown correctly.